.github/workflows/migrations.migrator.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Migrations
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: backend/migrator
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js 16.x
+      uses: actions/setup-node@v2
+      with:
+        node-version: 16
+    - name: Setup shared
+      working-directory: backend/shared
+      run: |
+        npm ci
+        npm run build
+    - name: Set up test postgres
+      run: docker run --name migrations-test -p 9999:5432 -e POSTGRES_USER=user -e POSTGRES_PASSWORD=password -e POSTGRES_DB=dsek -d postgres:14-alpine
+    - run: npm ci
+    - run: sleep 10
```

Comment:
Will this always work? Or can other factors make the sleep to short?

Reply:
It won't always work, but a more complex solution felt redundant, and if it fails too often, we can just add a couple of seconds